aspnetcore/signalr/Configuration.md

```diff
@@ -0,0 +1,124 @@
+---
+title: ASP.NET Core SignalR Configuration
+author: rachelappel
+description: Configure ASP.NET Core SignalR Apps
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.date: 06/30/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/configuration
+---
+
+# ASP.NET Core SignalR Configuration
+
+## [JSON]( https://github.com/aspnet/SignalR/blob/dev/src/Microsoft.AspNetCore.SignalR.Protocols.Json/JsonHubProtocolOptions.cs)/[MessagePack](https://github.com/aspnet/SignalR/blob/dev/src/Microsoft.AspNetCore.SignalR.Protocols.MessagePack/MessagePackHubProtocolOptions.cs) serialization options
+
+MessagePack is a binary serialization format that provides serialization options for server and client. 
+
+```csharp
+using Microsoft.Extensions.DependencyInjection
+
+var connection = new HubConnectionBuilder()
+    .WithUrl("...")
+    .AddJsonHubProtocol(options => {
+        // options.PayloadSerializerSettings is a JSON.NET JsonSerializerSettings used to 
+        // configure how the payload is serialized
+        // Example: PascalCase JSON (camelCase by default)
+        options.PayloadSerializerSettings.ContractResolver = new DefaultContractResolver();
+    })
+    .Build();
+```
+
+### [HubConnection](https://github.com/aspnet/SignalR/blob/dev/src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs#L66) options
+
+The following table describes the `HubConnection` options for configuring the client:
+
+| Option | Description |
+| ------ | ----------- |
+| `ServerTimeout` | If the server doesn't respond within this time interval, the connection closes |
+| `HandshakeTimeout`  | If the server doesn't respond to the initial handshake message within this time interval, the connection closes |
+| `PingInterval` (added in 2.2) | If the client hasn't sent a message within this interval, a ping message is sent automatically to keep the connection open |
```

Comment:
Let's remove this for now. Perhaps we should start a tracking issue for 2.2 changes?

Reply:
Filed: https://github.com/aspnet/Docs/issues/6925